docs/planning/prds/_index.md

```
@@ -27,20 +27,20 @@ updated: '2026-04-25'
 | [04b](./04b-sub-service-pages.md)       | Sub-Service Pages & Content Model    | completed   | high     | 2     | 12/12      | —                                         |
 | [04c](./04c-new-parent-services.md)     | New Top-Level Service Pages          | completed   | critical | 2     | 7/10       | —                                         |
 | [04d](./04d-location-pages.md)          | Location Pages                       | completed   | critical | 2-3   | 18/18      | —                                         |
-| [06](./06-blog-strategy.md)             | Blog Content Strategy                | in-progress | high     | 2-3   | 33/37      | [plan](../plans/06-blog-strategy/plan.md) |
-| [07](./07-geo-strategy.md)              | GEO (Generative Engine Optimisation) | approved    | medium   | 1-2   | 0/5        | —                                         |
+| [06](./06-blog-strategy.md)             | Blog Content Strategy                | approved    | high     | 2-3   | 4/5        | [plan](../plans/06-blog-strategy/plan.md) |
+| [07](./07-geo-strategy.md)             | GEO (Generative Engine Optimisation) | in-progress | medium   | 1-2   | 2/5        | [plan](../plans/07-geo-strategy/plan.md)  |
 | [08](./08-paid-ads.md)                  | Paid Ads Relaunch                    | draft       | medium   | 3     | 0/6        | —                                         |
 
 ## Pipeline Flow
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)